[xsp] add XSP classes back into Mono.WebServer.dll

This patch removes the XSP classes from the DLL

https://github.com/mono/xsp/commit/ff3098d7858060fc055abe276a6499e6dce43161https://github.com/mono/xsp/commit/ff3098d7858060fc055abe276a6499e6dce43161

However they are needed by monotools-server so until
a determination is made as to whether monotools-server
should be modified to remove the dependency on XSPWebSource
add these back into Mono.WebServer.dll

Upstream-status: Pending

Signed-off-by: Alex J Lennon <ajlennon@dynamicdevices.co.uk>

diff -ur git.org/src/Mono.WebServer/Mono.WebServer.sources git/src/Mono.WebServer/Mono.WebServer.sources
--- git.org/src/Mono.WebServer/Mono.WebServer.sources	2015-07-20 14:30:37.959761602 +0100
+++ git/src/Mono.WebServer/Mono.WebServer.sources	2015-07-20 14:31:31.475762349 +0100
@@ -51,3 +51,9 @@
 ./WebSource.cs
 ./WebTrace.cs
 ./Worker.cs
+../Mono.WebServer.XSP/XSPApplicationHost.cs 
+../Mono.WebServer.XSP/XSPWorkerRequest.cs 
+../Mono.WebServer.XSP/XSPRequestBroker.cs 
+../Mono.WebServer.XSP/XSPWebSource.cs 
+../Mono.WebServer.XSP/XSPWorker.cs 
+../Mono.WebServer.XSP/SslInformation.cs
